README.md

```diff
@@ -113,6 +113,196 @@ The below Environment variables are optional. If not provided a manual Applicati
 |`BOLD_SERVICES_USER_EMAIL`|Yes|It should be a valid email.|
 |`BOLD_SERVICES_USER_PASSWORD`|Yes|It should meet our password requirements.|
 
+## Environment variables for configuring `Branding` in backend
+The below Environment variables are optional. If not provided Bold BI will take the default configured values.
```

Comment:
The following environment variables are optional. If they are not provided, Bold BI will use the default configured values.

Reply:
concern addressed